src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -65,6 +157,97 @@ private void printIds() throws IOException {
     for (ServiceChannelType type : ServiceChannelType.values()) {
       printServiceChannelList(type, serviceChannelFiles);
     }
+    
+    System.out.println("\n    public final static String[][] ORGANIZATION_SERVICES = {");
+    List<String> organizationServices = new ArrayList<>();
+
+    for (int i = 0; i < organizationFiles.length; i++) {
+      File organizationFile = organizationFiles[i];
+      organizationServices.add(i, printOrganizationServicesList(i, organizationFile));
+    }
+
+    System.out.println(StringUtils.join(organizationServices, ",\n"));
+    System.out.println("    };");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "    };" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)